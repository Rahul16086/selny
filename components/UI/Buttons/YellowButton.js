import { Pressable, Text, StyleSheet } from "react-native";

const YellowButton = ({ onPress, children }) => {
  return (
    <Pressable
      android_ripple={{ color: "#ffffff" }}
      style={styles.buttonTouch}
      onPress={onPress}
    >
      <Text style={styles.fonts}>{children}</Text>
    </Pressable>
  );
};

const styles = StyleSheet.create({
  buttonTouch: {
    width: "100%",
    backgroundColor: "#FEB700",
    borderRadius: 5,
    height: 40,
    justifyContent: "center",
    alignItems: "center",
    marginBottom: 10,
  },
  fonts: {
    fontSize: 20,
    textTransform: "uppercase",
    fontWeight: "bold",
    justifyContent: "center",
    alignItems: "center",
  },
});

export default YellowButton;
