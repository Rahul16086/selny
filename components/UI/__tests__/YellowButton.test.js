import YellowButton from "../Buttons/YellowButton";
import React from "react";
import renderer from "react-test-renderer";

describe("Yellow Button", () => {
  it("renders properly", () => {
    const tree = renderer.create(<YellowButton />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
