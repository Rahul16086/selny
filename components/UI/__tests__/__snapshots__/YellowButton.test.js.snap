// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Yellow Button renders properly 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={true}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FEB700",
      "borderRadius": 10,
      "height": 40,
      "justifyContent": "center",
      "marginBottom": 10,
      "width": "100%",
    }
  }
>
  <Text
    style={
      Object {
        "fontFamily": "montserratSemiBold",
        "fontSize": 18,
        "textTransform": null,
      }
    }
  />
</View>
`;
